app/models/components/ability_host.rb

```diff
@@ -83,6 +83,17 @@ def currently_valid_hashes
         }
       ]
     end
+
+    # Returns a condition which will return started rows(start_at before current time) when
+    # ORed together in a database query. Reverse-merge this with your conditions to obtain the
+    # set of already started rows in the table.
+    #
+    # @return [Hash] The hash condition.
+    def already_started_hash
```

Comment:
I think it makes sense to show the old announcements when user intentionally visits the index page, but we do not actively display/notify the expired ones.